app/services/gov_uk/date_accessor.rb

```diff
@@ -0,0 +1,45 @@
+module GovUk
```

Comment:
add comment on what used for - basically to enable use of gov_uk_date_field helper on virtual attributes